README.md

```diff
@@ -43,11 +53,24 @@ maven {
 }
 ```
 
+**Native authentication:**
+
+
+```java
+dependencies {
+implementation 'com.microsoft.identity.client:msal:5.1.0'
+}
+maven {
+            url 'https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed/maven/v1'
+            name 'Duo-SDK-Feed'
+        }
+```
+
```

Comment:
This is required for integrating with MSAL and is not dependent on any flow.

Reply:
Thanks for the call out @iamgusain. I'm updating it.